integration-tests/utils/integration-utils.js

```diff
@@ -76,17 +78,25 @@ const signIntoTerra = async page => {
   return page.evaluate(token => window.forceSignIn(token), bearerToken)
 }
 
+const waitForElement = (page, xpath) => {
+  return page.waitForXPath(xpath)
+}
+
+const svgText = ({ textContains }) => {
```

Comment:
`textContains` should be consistent with whatever we choose to do with `clickable`.